constitution.md

```diff
@@ -0,0 +1,75 @@
+# Queer Code Scotland: Constitution
+
+## Name
+
+The name of the organisation is Queer Code Scotland.
+
+## Purpose
+
+To create an inclusive and welcoming space for LGBTQ+ technologists by organising meetups in 
+Scotland.
+
+## Code of conduct
+
+Our meetups are open to anyone who would like to join. We expect all attendees to adhere to the 
+following code of conduct. https://queer-code.org/coc
+
+Anyone asked to stop unacceptable behavior is expected to stop immediately. If a community member 
+engages in unacceptable behavior, the community organisers may take any action they deem 
+appropriate, up to and including a temporary ban or permanent expulsion from the community without 
+warning (and without refund in the case of a paid event).
+
+## Membership
+
+A member is deemed as someone who is a member of our meetup group. 
+Membership is open for people to join and leave at any time. https://www.meetup.com/qcscot/members/
+
+## Subscriptions
+
+There is no cost of membership. On occasion we may decide to charge an 
+entry fee if we have to incur costs in order to put on an event, however we will endeavour to avoid 
+this wherever possible. 
+
+## Committee
+
+The committee exists primarily for the management of our finances. Other day to day activities are
+organised by the volunteers on #organisation-scotland slack channel on the queer code slack. 
```

Comment:
Suggest changing “volunteers on” to “volunteers on the”, “slack” to “Slack”, and “queer code slack” to “Queer Code Slack”.